lib/inspector.coffee

```diff
@@ -50,7 +56,7 @@ class Inspector
             cursor_pos = cursor.getBufferColumn()
         return [code, cursor_pos]
 
-    getInspector: ->
+    initializeInspector: ->
```

Comment:
On 11/07/16 11:40, Lukas Geiger wrote:
> In lib/inspector.coffee <https://github.com/nteract/hydrogen/pull/338#discussion_r70235799>:
> 
>> @@ -50,7 +56,7 @@ class Inspector
>>              cursor_pos = cursor.getBufferColumn()
>>          return [code, cursor_pos]
>>  
>> -    getInspector: ->
>> +    initializeInspector: ->
> 
> I'm OK with renaming |@inspector| to |@view|. But I don't think |@initializeInspector| should be renamed to |@show| since it just constructs or clears the view. The inspector is only shown if a message is added.
> 
> I didn't move the creation to the constructor because the |inspect| command is only used on occasions, hence I think the view should only be created on the first |inspect| request. Or am I missing something?

I find confusing that an `init` function is invoked multiple times.

I think the real problem is that `@initializeInspector` is used to accomplish two different goals: construct the view and clear the view.

Since the view is to be constructed on first call to `@toggle`, why not having a check in `@toggle` to create `@view`:

```
@view ?= new MessagePanelView, title: 'Hydrogen Inspector'
```

and replace the call to `@initializeInspector` with a `@view.clear()` ?